evm/src/main/java/org/hyperledger/besu/evm/MainnetEVMs.java

```diff
@@ -468,4 +469,15 @@ public static void registerShandongOperations(
     registry.put(new RetFOperation(gasCalculator));
     registry.put(new JumpFOperation(gasCalculator));
   }
+
+  public static EVM shandong(final BigInteger chainId, final EvmConfiguration evmConfiguration) {
```

Comment:
Do we need to keep Shandong's EVM and spec?

Reply:
I have another PR where I nuke it from orbit.  I wanted to keep them separate in case we need to roll this one out.